packages/swapper/src/swappers/ChainflipSwapper/endpoints.ts

```diff
@@ -0,0 +1,376 @@
+        `&destinationAddress=${tradeQuote.receiveAddress}` +
+        `&boostFee=10` +
+        `&minimumPrice=${minimumPrice}` +
+        `&refundAddress=${from}` +
+        `&retryDurationInBlocks=10` +
+        `&commissionBps=${serviceCommission}`,
+
+      // TODO: For DCA swaps we need to add the numberOfChunks/chunkIntervalBlocks parameters
+    )
+
+    if (maybeSwapResponse.isErr()) {
+      const error = maybeSwapResponse.unwrapErr()
+      const cause = error.cause as AxiosError<any, any>
+      throw Error(cause.response!.data.detail)
+    }
+
+    const { data: swapResponse } = maybeSwapResponse.unwrap()
+
+    if (!swapResponse.id) throw Error('missing swap ID')
+
+    tradeQuoteMetadata.set(tradeQuote.id, swapResponse)
+
+    const depositAddress = swapResponse.address!
+    const { assetReference } = fromAssetId(step.sellAsset.assetId)
+    const isTokenSend = isToken(step.sellAsset.assetId)
+
+    const adapter = assertGetEvmChainAdapter(step.sellAsset.chainId)
+
+    const getFeeDataInput: GetFeeDataInput<EvmChainId> = {
+      to: depositAddress,
+      value: step.sellAmountIncludingProtocolFeesCryptoBaseUnit,
+      chainSpecific: {
+        from,
+        contractAddress: isTokenSend ? assetReference : undefined,
+        data: undefined,
+      },
+      sendMax: false,
+    }
+    const feeData = await adapter.getFeeData(getFeeDataInput)
+    const fees = feeData[FeeDataKey.Average]
+
+    if (!isExecutableTradeStep(step)) throw Error('Unable to execute trade step')
+
+    const unsignedTxInput = await adapter.buildSendApiTransaction({
+      to: depositAddress,
+      from,
+      value: step.sellAmountIncludingProtocolFeesCryptoBaseUnit,
+      accountNumber: step.accountNumber,
+      chainSpecific: {
+        gasLimit: fees.chainSpecific.gasLimit,
+        contractAddress: isTokenSend ? assetReference : undefined,
+        ...(supportsEIP1559
+          ? {
+              maxFeePerGas: fees.chainSpecific.maxFeePerGas!,
+              maxPriorityFeePerGas: fees.chainSpecific.maxPriorityFeePerGas!,
+            }
+          : {
+              gasPrice: fees.chainSpecific.gasPrice,
+            }),
+      },
+    })
+
+    return {
+      chainId: Number(fromChainId(chainId).chainReference),
+      data: unsignedTxInput.data,
+      to: unsignedTxInput.to,
+      from,
+      value: unsignedTxInput.value,
+      gasLimit: unsignedTxInput.gasLimit,
+      maxFeePerGas: unsignedTxInput.maxFeePerGas,
+      maxPriorityFeePerGas: unsignedTxInput.maxPriorityFeePerGas,
+      gasPrice: unsignedTxInput.gasPrice,
+    }
+  },
+  getUnsignedUtxoTransaction: async ({
+    tradeQuote,
+    xpub,
+    accountType,
+    assertGetUtxoChainAdapter,
+    config,
+  }: GetUnsignedUtxoTransactionArgs): Promise<BTCSignTx> => {
+    if (!isExecutableTradeQuote(tradeQuote)) throw Error('Unable to execute trade')
+
+    const brokerUrl = config.REACT_APP_CHAINFLIP_API_URL
+    const apiKey = config.REACT_APP_CHAINFLIP_API_KEY
```

Comment:
Can't do that @NeOMakinG since this is relying on `config` which is *injected* for each handler, but not available in outer scope (this is swapper, not web so can't `getConfig()` here)

Reply:
> Can't do that @NeOMakinG since this is relying on `config` which is _injected_ for each handler, but not available in outer scope (this is swapper, not web so can't `getConfig()` here)

Ah shit, I didn't notice it was from an injected dependency...